next.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  env: {
+    NEXT_PUBLIC_BUILD_DATE: process.env.NEXT_PUBLIC_BUILD_DATE,
+    NEXT_PUBLIC_TC_BUILD: process.env.NEXT_PUBLIC_TC_BUILD,
```

Comment:
Is it possible to set defaults here or does it matter?

Reply:
we need to add 2 variables to the env file :)

Reply:
you can add the variables (or not) it doesn't break or do anything if they dont exist, these variables are mainly useful for deployed builds to see the build date and TC build id in the source at a quick reference. :)